packages/dd-trace/test/guardrails/telemetry.spec.js

```diff
@@ -8,7 +8,7 @@ describe('sendTelemetry', () => {
   let cleanup, sendTelemetry
 
   beforeEach(() => {
-    cleanup = telemetryForwarder()
+    cleanup = telemetryForwarder(true)
```

Comment:
This is already defaulted to `true`, am I missing something?

https://github.com/DataDog/dd-trace-js/blob/57efb0af7991982ef47316bcb9701003f5b2954d/integration-tests/helpers/index.js#L243